dev/Common/TerminalVelocityFeatures-PushNotifications.h

```diff
@@ -3,21 +3,21 @@
 
 // THIS FILE IS AUTOMATICALLY GENERATED; DO NOT EDIT IT
 
-// INPUT FILE: dev\common\TerminalVelocityFeatures-DeploymentAPI.xml
+// INPUT FILE: dev\common\TerminalVelocityFeatures-PushNotifications.xml
 // OPTIONS: -Channel Experimental -Language C++ -Namespace Microsoft.Windows.PushNotifications
 
 // Feature constants
-#define WINDOWSAPPRUNTIME_MICROSOFT_WINDOWS_PUSHNOTIFICATIONS_FEATURE_DEPLOYMENTAPI_ENABLED 1
+#define WINDOWSAPPRUNTIME_MICROSOFT_WINDOWS_PUSHNOTIFICATIONS_FEATURE_PUSHNOTIFICATIONS_ENABLED 1
 
 #if defined(__cplusplus)
 
 namespace Microsoft::Windows::PushNotifications
 {
 
-__pragma(detect_mismatch("ODR_violation_WINDOWSAPPRUNTIME_MICROSOFT_WINDOWS_PUSHNOTIFICATIONS_FEATURE_DEPLOYMENTAPI_ENABLED_mismatch", "AlwaysEnabled"))
-struct Feature_DeploymentAPI
+__pragma(detect_mismatch("ODR_violation_WINDOWSAPPRUNTIME_MICROSOFT_WINDOWS_PUSHNOTIFICATIONS_FEATURE_PUSHNOTIFICATIONS_ENABLED_mismatch", "AlwaysEnabled"))
```

Comment:
Does this mean that by default, our experimental APIs are going to just work here? If so, any reason for having a velocity check in the first place?

Reply:
In the pipeline, a script "Generate-TerminalVelocityFeatures.cmd" will create this header file and the constant WINDOWSAPPRUNTIME_MICROSOFT_WINDOWS_PUSHNOTIFICATIONS_FEATURE_PUSHNOTIFICATIONS_ENABLED, will be set depending on the -Channel flag passed. This .h was created with the 'Experimental' flag but will be overrode in the pipeline with 'Preview/Stable' flags where a different header file will be made.

Here is a link to some comments from @DrusTheAxe  about it:
https://github.com/microsoft/WindowsAppSDK/pull/1403#discussion_r707576162